polaris/ocean/tasks/geostrophic/geostrophic.cfg

```diff
@@ -0,0 +1,125 @@
+# Options related to the vertical grid
+[vertical_grid]
+
+# the type of vertical grid
+grid_type = uniform
+
+# Number of vertical levels
+vert_levels = 1
+
+# Depth of the bottom of the ocean
+bottom_depth = 2000.0
+
+# The type of vertical coordinate (e.g. z-level, z-star)
+coord_type = z-star
+
+# Whether to use "partial" or "full", or "None" to not alter the topography
+partial_cell_type = None
+
+# The minimum fraction of a layer for partial cells
+min_pc_fraction = 0.1
+
+
+# config options for spherical convergence tests
+[spherical_convergence]
```

Comment:
Do we want to add `icos_resolutions` and `qu_resolutions` here?

Reply:
I think it's a good idea to use the default resolutions from:
https://github.com/E3SM-Project/polaris/blob/4bc796e561f8fe1e31c09e34c4abda793d649d6d/polaris/ocean/convergence/spherical/spherical.cfg#L1-L8
That way if we decide later to change the resolutions there, the shared steps in `geostrophic` (and in `cosine_bell`) get updated the same way.  But if you have a reason you think `geostrophic`, `cosine_bell` and others should specify their own, possibly unique list of resolutions, I'm keen to hear it.